Assingmenty/DiceRollGame/MessagePrinter.cs

```diff
@@ -0,0 +1,23 @@
+﻿class MessagePrinter
+{
+    public static void WelcomeUser(int numberOfTries)
+    {
+        Console.WriteLine($"Dice rolled. Guess what number it shows in {numberOfTries.ToString()} tries.");
+    }
+
+    public static void AskForUserInput()
+    {
+        Console.WriteLine("Enter a number: ");
+    }
+
+    public static void PrintGameResult(bool isWon)
+    {
+        if (isWon) { Console.WriteLine("You win."); }
```

Comment:
Raczej zawsze używa się nawiasów klamrowych tak, że każdy jest w osobnej linijce. Jak cię tu boli że taka prosta metoda ma tyle linijek, zawsze możesz użyć operatora trójargumentowego o którym chyba jeszcze nie było lekcji:

var message = isWon ? "You win" : "You lose";
Console.WriteLine(message);

Reply:
Nie było, ale w Pytonku go lubię, więc tu też będę stosować, dzięki.